Dockerfile

```diff
@@ -127,3 +128,14 @@ RUN chmod +x /tools/obodash && \
 
 # Install KGCL ROBOT plugin
 RUN wget -nv -O /tools/robot-plugins/kgcl.jar https://github.com/gouttegd/kgcl-java/releases/download/kgcl-java-$KGCL_JAVA_VERSION/kgcl-robot-plugin-$KGCL_JAVA_VERSION.jar
+
+RUN if [ "$TARGETARCH" = "amd64" ]; then \
+        wget -nv https://github.com/mikefarah/yq/releases/download/v$YQ_VERSION/yq_linux_amd64 -O /tools/yq && \
```

Comment:
Annoyingly enough, there is also a [Python package also called yq](https://github.com/kislyuk/yq), which also installs an executable called `yq`. That package is _not_ provided in the ODK, but still I don’t like the risk of confusion (e.g. if someone installs the Python package in a custom workflow, they would probably expect that a call to `yq` invokes the Python script rather than this `yq` ).

Could we rename the binary to something like `mf-yq` (Mike Farrah’s `yq`), `yq-go` (`yq` in GO), or similar?

Reply:
Makes sense, did!